config/routes.rb

```diff
@@ -1,5 +1,7 @@
-Rails.application.routes.draw do
```

Comment:
Great, I have a few more commits to add to this for your review, and then I'll add tests for you.